### ISSUE TITLE
Can't import users from a user group - "Asset Id cannot be blank."

### ISSUE THREAD

**Issue by rossthomson89** [putyourlightson/craft-campaign#289]
### The Issue
Hi there,

I'm suddenly having import issues after upgrading the plugin from v1.20.0 to v1.22.2. I'm trying to import a new group of users setup within Craft CMS and this feature used to work flawlessly, but now every time I try a new import I get a "Couldn't import user group" error message from the dashboard. I'm seeing this issue across all development environments (local, staging & production).

Upon further investigation I found this error message within the campaign.log file:
**2022-01-19 13:32:23 [][1][info] Couldn’t import user group. Asset Id cannot be blank.**

We have never used (or created) a custom user field for "Asset Id" at any point of this project - so I'm not entirely sure what "Asset Id" is blank? At a loss on how to debug this so any advice or support would be much appreciated.

### To reproduce
Steps to reproduce the behaviour:
1. Create a Mailing List
2. Go to "Contacts" 
3. Select your newly created Mailing List and click "Import"
4. Select ANY user group and click "next"
5. Select you're newly created Mailing List and click "Import"
6. Import fails with message "Couldn't import user group"

### Versions
- Plugin version: 1.22.2
- Craft version: 3.7.29

Many Thanks
Ross

**Comment by bencroker** [comment]
Hi Ross,

The error sounds like it perhaps has to do with the temporary asset upload location that is used during the import, although this was added in version 1.17.3 (https://github.com/putyourlightson/craft-campaign/blob/develop/CHANGELOG.md#1173---2020-12-09).

> Importing contact data now stores CSV files in the temporary asset upload location as defined in _Settings → Assets → Settings_.

Can you please ensure that the "Temp Uploads Location" is defined and is writeable?

**Comment by rossthomson89** [comment]
Hi Ben,

Thanks for getting back to me!

I've checked the temporary asset upload location in Settings → Assets → Settings and it's set to "In the local temp folder". 

I then uploaded a random image to Assets → Temporary Uploads and it uploaded successfully. This newly uploaded image to the temp folder is stored in "/storage/runtime/assets/tempuploads/user_xxxx/nameOfRandomImage.png"

So as far as I can tell the folder  "Temp Uploads Location" is defined and is writeable. Do I need to create a custom temp uploads folder in the /public folder in order to fix this? I almost always use Amazon S3 to store all of our client assets so the temp uploads folder is usually something we pretty much ignore. 

Thanks
Ross
